packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -124,8 +131,17 @@ export class VaRadioOption {
               id={id + 'input'}
             />
           <label htmlFor={id + 'input'}>
-            {label}
-            {description && <span class="description" aria-describedby="option-label">{description}</span>}
+            <div>
```

Comment:
Should a `div` go inside of a label? Would it be more semantically correct to make this a span and apply block styling to it?

Reply:
I changed it to a `span`, but it shouldn't need block styling since it's targeted by display flex.